portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -72,8 +72,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
   @Override
   public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {
-      System.err.println("Path link is: " + request.getRequestURL());
-    long id = Long.parseLong(request.getParameter("id"));
+    String idString = request.getPathInfo().substring(1);
```

Comment:
What does "pathInfo" return? Is it everything after "/comment"?  i.e. /comment/2 -> /2?  Can you use String.split here and split based on "/"? Then we'd just return 2. In the code you have, if for some reason we put it /comment/2/, it would get "2/".

 https://docs.oracle.com/javase/7/docs/api/java/lang/String.html

Reply:
pathInfo returns "/[id#]", using String.split is a better alternative than String.substring like what you've stated. Done